python/openassetio/managerAPI/ManagerInterface.py

```diff
@@ -27,13 +27,15 @@
 # pylint: disable=unused-argument,no-self-use
 
 import abc
+# TODO(DF): Remove pylint disable once CI is fixed.
+import _openassetio  # pylint: disable=import-error
```

Comment:
[consider] This looks like a good candidate for a relative import!? (unless I'm missing something)

Reply:
The directory structure looks like
```
.
└── site-packages
    ├── openassetio
    │   ├── managerAPI
    │   │   ├── __init__.py
    │   │   ├── ManagerInterface.py
    ├── _openassetio.so
```
i.e. the `_openassetio` Python extension module is installed at the root of `site-packages`, not within the same package, so can't be relatively imported. 

Does this seem reasonable?

Reply:
Oh I see! Sorry my bad, I thought it was all part of the same package. In that case it makes perfect sense :+1:

Reply:
Question, would it be worth moving it under `openassetio`?

Reply:
Moved under `openassetio` in 9b48c9ac, which necessitated a full `pip install` without `--editable`, updated in 025f4a7c.